documents/motions/10.2020.0 - Committees and Teams.md

```diff
@@ -14,6 +14,7 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
 2. All WCA Committees and Teams shall be formed for an indefinite period, unless otherwise decided by the WCA Board of Directors.
 3. Each WCA Committee and Team will have a direct relationship with at least one Director for management, communication, and escalation.
 4. Each WCA Committee and Team shall have one Leader.
+   0. Exception: The WCA Executive Assistants Team is led by the WCA Board.
    1. The Leader of every WCA Committee and Team is chosen by the WCA Board of Directors, based on a voting in the WCA Board of Directors with at least supermajority, after a request for applications among the Community.
```

Comment:
```suggestion
   2. The Leader of every WCA Committee and Team is chosen by the WCA Board of Directors, based on a voting in the WCA Board of Directors with at least supermajority, after a request for applications among the Community.
```

Reply:
Also 4.2 -> 4.3 and 4.3 -> 4.4, but I don't see a way to do that in the online editor for this PR.

Reply:
You should be able to edit this from GitHub if you open the entire document instead of trying to edit it from the PR. I can do a PR with the renumbering if you want.

Reply:
Upcoming amendments fixes this.